misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/JBallerinaDebugServer.java

```diff
@@ -1105,6 +1109,8 @@ private void attachToRemoteVM(String hostName, int portName) throws IOException,
         EventRequestManager erm = context.getEventManager();
         ClassPrepareRequest classPrepareRequest = erm.createClassPrepareRequest();
         classPrepareRequest.enable();
```

Comment:
Just to make be consistent
```suggestion
        ClassPrepareRequest classPrepareRequest = erm.createClassPrepareRequest().enable();
```